koku/masu/external/kafka_msg_handler.py

```diff
@@ -546,29 +549,39 @@ def process_report(report):
     complete.
 
     Args:
+        request_id (Str): The request id
         report (Dict) - keys: value
+                        request_id: String,
+                        account: String,
                         schema_name: String,
                         manifest_id: Integer,
                         provider_uuid: String,
                         provider_type: String,
                         current_file: String,
+                        date: DateTime
 
     Returns:
         True if line item report processing is complete.
 
     """
+    account = report.get("account")
     schema_name = report.get("schema_name")
     manifest_id = report.get("manifest_id")
     provider_uuid = report.get("provider_uuid")
     provider_type = report.get("provider_type")
+    start_date = report.get("date")
 
     report_dict = {
         "file": report.get("current_file"),
         "compression": UNCOMPRESSED,
         "manifest_id": manifest_id,
         "provider_uuid": provider_uuid,
     }
-    return _process_report_file(schema_name, provider_type, provider_uuid, report_dict)
+    result = _process_report_file(schema_name, provider_type, provider_uuid, report_dict)
+    if start_date:
+        start_date_str = start_date.strftime("%Y-%m-%d")
+        convert_to_parquet.delay(request_id, account, provider_uuid, provider_type, start_date_str, manifest_id)
```

Comment:
So `convert_to_parquet` is done on the manifest level right?  Does it make sense to do this after summary (in `summarize_manifest`) since then we would know that all report files have been uploaded and we do this just one time?

Reply:
Here's an example of how we are using a part of the downloader in the real OCP-ingress service flow:
https://github.com/project-koku/koku/blob/368f317a025b905210af92c3cddfeeca5bf143dc/koku/masu/external/kafka_msg_handler.py#L109

Once the worker scaling changes are in I think we can come back to this and clean it up...